src/style.css

```
@@ -1 +1,31 @@
-/* styles go here */
+div.personalinfo {
+  margin-top: 10px;
+}
+
+fieldset.personalinfo {
+  margin-bottom: 20px;
+}
+
+fieldset.registration {
+  margin-bottom: 20px;
+}
+
+fieldset.facts {
+  margin-bottom: 20px;
+}
```

```suggestion
.personalinfo,
.registration,
.facts {
  margin-bottom: 20px;
}
```
скомбінуй селектори з однаковими стилями, або краще зроби їм спільний клас